ResearchKit/Common/ORKDefines_Private.h

```diff
@@ -37,8 +37,9 @@ ORK_EXTERN NSBundle *ORKBundle() ORK_AVAILABLE_DECL;
 #define ORKLocalizedString(key, comment) \
 [ORKBundle() localizedStringForKey:(key) value:@"" table:nil]
 
-#define ORKDynamicCast(x, c) ((c *) ([x isKindOfClass:[c class]] ? x : nil))
+ORK_EXTERN id _ORKDynamicCast(id x, Class objClass) ORK_AVAILABLE_DECL;
 
+#define ORKDynamicCast(x, c) (c *) _ORKDynamicCast(x, [c class])
```

Comment:
Let's move this to ORKHelpers.h

We don't want this to live in API (and anything _Private.h is exposed in the private headers and so is developer accessible, even if it is not Swift accessible).